### DISCUSSION TITLE
How to promote `export not found` warnings to errors?

### DISCUSSION THREAD

**Discussion by Emilios1995** [webpack/webpack#13697] [Q&A]
Was there ever a Webpack version where importing exports that didn't exist resulted in an error? Right now they show up as warnings. Is there a way of promoting them to errors?

**Comment by alexander-akait** [comment]
Better to fix it, but we have `ignoreWarnings` option

**Reply by alexander-akait** [reply]
https://webpack.js.org/configuration/other-options/#ignorewarnings

**Reply by Emilios1995** [reply]
Hmm but I don't want to ignore it, I want to get it as an error instead of a warning.
